mono/mini/aot-runtime.c

```diff
@@ -3313,9 +3315,11 @@ mono_aot_find_jit_info (MonoDomain *domain, MonoImage *image, gpointer addr)
 
 	nmethods = amodule->info.nmethods;
 
+#ifndef ENABLE_INTERPRETER
```

Comment:
yes, domains won't work with mixed mode (interp + full-aot).

the scenario is interp + domains, and that requires some aot-ed wrappers.